<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TorinoScan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,1,0,0">
    <item row="3" column="0" colspan="5">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabParameters">
       <attribute name="title">
        <string>Parameters</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="enableAllParamButton">
          <property name="text">
           <string>Enable all</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="disableAllParamButton">
          <property name="text">
           <string>Disable all</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QScrollArea" name="parametersScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="parametersCentralWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>700</width>
             <height>209</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="parametersGrid"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFaultcodes">
       <attribute name="title">
        <string>Fault codes</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="readFaultCodesButton">
          <property name="text">
           <string>Read fault codes</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QScrollArea" name="faultCodesScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="faultCodesCentralWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>700</width>
             <height>209</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="faultCodesGrid"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="actuatorsTab">
       <attribute name="title">
        <string>Actuators</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QScrollArea" name="actuatorsScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="actuatorsCentralWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>700</width>
             <height>241</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="actuatorsGrid"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="ecuDefinitionLabel">
      <property name="text">
       <string>ECU definition:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ftdiDevicesLabel">
      <property name="text">
       <string>FTDI devices:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="carComboBox"/>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QComboBox" name="ftdiDeviceComboBox"/>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="ecuComboBox"/>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="disconnectButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLabel" name="statusLabel">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Status: not connected</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Set definition</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
